README.md

```diff
@@ -112,10 +112,10 @@ For a nice visualization of a crystal, show
 from weas_widget import WeasWidget
 viewer1 = WeasWidget()
 viewer1.load_example("tio2.cif")
-viewer1.modelStyle = 2
-viewer1.boundary = [[-0.1, 1.1], [-0.1, 1.1], [-0.1, 1.1]]
-viewer1.showBondedAtoms = True
-viewer1.colorType = "VESTA"
+viewer1.avr.model_style = 2
```

Comment:
just pointing out that the meaning of `avr` is not intuitively clear to me

Reply:
It's `AtomsViewer` :laughing: . I am thinking of three names:
- `viewer.avr.model_style=1`
- `viewer.atoms_viewer.model_style=1`
- `viewer.atoms.model_style=1`
Which one do you think is better? Or any other names?

BTW, tor the moment, the widget has three top-level attributes:
- `avr`: AtomsViewer,
- `camera`
- `imp`: InstancedMeshPrimitive
In the future, we will add
- `light`
- `ops`: Operators, e.g., `viewer.ops.objects.delete()`
- `scene`, access all objects in the scene.

Reply:
I would vot for the option `viewer.atoms` in that case

camera, scene and light are self-explanatory.
`imp` is not - perhaps `mesh`?
`ops` is also not so clear to me - will there be other properties besides `objects`? (if not, `objects` could be a top-level attribute)

Reply:
Thanks! I need to think about this more; Most names come directly from the JavaScript part. I need to make sure the names are good on both sides.